erts/emulator/beam/big.h

```diff
@@ -37,8 +37,11 @@ typedef Uint16   ErtsHalfDigit;
 #undef  BIG_HAVE_DOUBLE_DIGIT
 typedef Uint16   ErtsHalfDigit;
 
+#elif (SIZEOF_VOID_P == 8) && defined(__GNUC__) && (__GNUC__ >= 4)
+typedef __uint128_t ErtsDoubleDigit;
```

Comment:
Is this universally available on all 64-bit targets?

Reply:
It seems to be. I tried all 64-bit gcc and clang targets I could find on godbolt.org.